app/tree/gui/src/ts/twxplore/gui/geo/components/geometry/block.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
```

Comment:
Block (capitalized)

Reply:
Maybe BlockList? I usually reserve unsuffixed names like Block for the model, and suffix the component name with some hint as to what it is (list, table, button, whatever).